rust/src/mask/config/mod.rs

```diff
@@ -164,16 +164,18 @@ pub struct MaskConfig {
 }
 
 impl MaskConfig {
-    /// Return the number of bytes needed to represent a mask element
-    /// with this configuration
-    pub fn bytes_per_digit(&self) -> usize {
+    /// Returns the number of bytes needed for an element of a mask or masked model wrt this mask
+    /// configuration.
+    pub fn bytes_per_number(&self) -> usize {
         let max_number = self.order() - BigUint::from(1_u8);
         (max_number.bits() + 7) / 8
     }
 
+    /// Gets the additional shift value for masking/unmasking wrt this mask configuration.
```

Comment:
nit: "wrt this" seems a bit heavy to me. This method belongs to `MaskConfig` so maybe we don't even need to explicitly say it:

> Gets the additional shift value for masking/unmasking

If we want to keep it, maybe we can just use "with"

> Gets the additional shift value for masking/unmasking with this configuration

Reply:
removed the redundancy.